src/popup/pages/TipsClaim.vue

```diff
@@ -83,6 +84,7 @@ export default defineComponent({
     const { t } = useI18n();
     const store = useStore();
     const router = useRouter();
+    const api = useAeApiActions();
```

Comment:
I suggest to treat this a a regular composable, so pick the methods from returned object instead of creating the `api` constant.